### ISSUE TITLE
`operator.get_conn` inconsistency

### ISSUE THREAD

**Issue by PhilipVinc** [netket/netket#316]
according to [`py_operator.hpp`](https://github.com/netket/netket/blob/1bd16b3c4b962fed134aa371ead2f8e5dcceb80e/Sources/Operator/py_operator.hpp#L50), `operator.get_conn(v)` in python should return the new states, instead it returns (and is tested to return) the delta of the configurations...

We should fix the docs

**Comment by gcarleo** [comment]
Oups, I guess then we should change the behavior of get_conn to match the docs actually. 

In any case, the idea for the future python interface would be to have a generator-like behavior 

```python
for state,mel in op[v]:
   do_something_with(state,mel)
```  
and also use it like a regular matrix: 

```python
mel=op[v,vprime]
```

**Comment by PhilipVinc** [comment]
Closed by #321
